### PR TITLE
Corsa monitoring changes

### PR DESCRIPTION
**Pull request by mcevik0**
Additional functions for submitting REST calls to the switches for pulling monitoring info.

### CODE REVIEW

File: networking_generic_switch/devices/corsa_devices/corsavfc.py [hunk 1]
```diff
@@ -451,6 +447,555 @@ def bridge_detach_tunnel(headers,
...
+#
+# GET EQUIPMENT
+#
+#   200 OK
+#
+def get_equipment(headers,
+                  url_switch):
+    url = url_switch + ep_equipment
+
+    try:
```
Comment (diurnalist) [inline networking_generic_switch/devices/corsa_devices/corsavfc.py]: General comment: if you are doing `except... raise`, there is no reason to have a `try` in the first place. The reason is that the exception that is caught would immediately be re-raised without being handled, just as if you did not attempt to catch it at all. Most of these functions then become one-liners:

```python
def get_equipment(headers, url_switch):
    return requests.get(url_switch + ep_equipment, headers=headers, verify=False)
```

File: networking_generic_switch/devices/corsa_devices/corsavfc.py [hunk 2]
```diff
@@ -451,6 +447,555 @@ def bridge_detach_tunnel(headers,
...
+# GET DATAPATH
+#
+#   200 OK
+#   303 See Other
+def get_datapath(headers,
+                 url_switch):
+    url = url_switch + ep_datapath
+
+    try:
+        r = requests.get(url, headers=headers, verify=False)
```
Comment (diurnalist) [inline networking_generic_switch/devices/corsa_devices/corsavfc.py]: Actually.. why all the `verify=False`? That's a bit odd.

Comment (mcevik0) [reply networking_generic_switch/devices/corsa_devices/corsavfc.py]: Switches have a self-signed cert on them that is used to enable REST server.

**Comment by diurnalist** [comment]
@mcevik0 I have actually integrated this to a separate piece of code: https://github.com/ChameleonCloud/low_power_exec/blob/corsa-monitoring/collect_corsa.py - I don't think we really need this hanging around in the NGS code as its job is coordinating the switch. If we wanted we could extract this into a corsa library later and share the code between the two applications. I think for now though, we can close this, as the work has been effectively moved.

**Pull request closed without merging**